tests/dummy/app/styles/app.css

```diff
@@ -0,0 +1 @@
+body { background-color: #caf0ff; }
```

Comment:
Is this just to have some test CSS in the dummy app?

Reply:
Yeah, IIRC `stylelint` complained that the CSS file was empty.